source/creatureeffect/CreatureEffectSpeedChange.cpp

```diff
@@ -59,6 +59,7 @@ void CreatureEffectSpeedChange::applyEffect(Creature& creature)
 
 void CreatureEffectSpeedChange::releaseEffect(Creature& creature)
 {
+    //FIXME: sanitizer reports that this is called at least once on something that is not a creature.
```

Comment:
Btw, this is cool to see things being fixed, but is it a good idea to add fixme everywhere, even for things you know are false positives? Shouldn't this be an issue instead?